chalicelib/checks/wrangler_checks.py

```diff
@@ -1176,7 +1176,8 @@ def users_with_doppelganger(connection, **kwargs):
             cases.append(log)
         # if not, compare names
         else:
-            score = fuzz.token_sort_ratio(us1['display_title'], us2['display_title'])
+            matcher = stringmatch.Levenshtein()
+            score = matcher.get_sim_score(us1['display_title'], us2['display_title']) * 100
```

Comment:
I assume the `* 100` means it's now a float 0.0-1.0. Was it previously a float at higher scale, or was it an integer? This will affect:
* whether 85.12345 if rounded to the nearest integer is greater than 85, because `score > 85` below will be true for `85.12345` but not for `85`.
* whether the part below where you use `'log': 'has similar names ({}/100)'.format(str(score))` and whether it shows a possibly very long decimal part.

Reply:
I added a round().  The actual cutoff is arbitrary so rounding should not significantly affect anything but will potentially tidy up output.